### DISCUSSION TITLE
windows extension?

### DISCUSSION THREAD

**Discussion by mmavka** [timescale/timescaledb-toolkit#447] [Q&A] [answered]
All hello!
Can't build project on windows 11. I get compiler error error[E0412]: cannot find type `pid_t` in crate `libc` (cargo-pgx).
It builds on ubuntu with no problems.
maybe there is a ready extension for pg14 for win?

**Comment by jerryxwu** [comment, marked as answer]
@mmavka Windows is not a platform that we test or support. One of the obstacles is PGX (which the Toolkit depends heavily on) does not support Windows.
